build.sbt

```diff
@@ -91,7 +91,7 @@ val root = (project in file("."))
       "dev.zio"                %% "zio"                         % zioVersion,
       "dev.zio"                %% "zio-interop-reactivestreams" % zioRSVersion,
```

Comment:
Please make ZIO, zio-interop-reactivestreams and Slick as Provided, by adding the `% Provided` tag